jvm/ml4ir-inference/src/test/integration/ml4ir/inference/tensorflow/TensorFlowInferenceIT.scala

```diff
@@ -53,11 +89,109 @@ class TensorFlowInferenceIT extends TestData {
     return Source.fromFile(csvFile).getLines.toList.tail.flatMap(extractColumnValues)
```

Comment:
no need for return keyword in scala (last value in an expression / method is the "return value" of it)

Reply:
👍